### ISSUE TITLE
[UI-FIX] auto fill popup for pin code appears

### ISSUE THREAD

**Issue by georgemac70cr8** [asgiom/cr8-brm-mobile-issues#50]
**Section or interface element**
Log In using pin code... "auto fill pop up appears" 

I don't know if this is android / huawaei / google security - but when I log in using the pin an annoying auto fill popup appears. and, as far as i can see, is completely useless and does nothing (because it's function is for sms / email verification i would imagine). in fact, when i press it, it actually means i can no longer input the pin and need to close and reopen app.

screenshot attached. Please try recreating first on the huawei you have there.

![Screenshot_20200901_025756_com opibis cr8 mobile](https://user-images.githubusercontent.com/69824025/91825191-36372d00-ec3c-11ea-825f-d8294d36c74a.jpg)

**Comment by t3nshi** [comment]
Yes I could reproduce this on one of my phones. I will see if there is something we can do about it, since it is not the app that does this, but the Os

**Comment by t3nshi** [comment]
This was a bit tricky and many devs are complaining to google because it seems that the workarounds always get override by google on new versions of android.. but I tested on the phones that showed me this and now it does not appear. Please let me know when you have a chance to check

**Comment by whywefight** [comment]
The button stays selected till you tap elsewhere still.

**Comment by t3nshi** [comment]
Can you confirm that you no longer have this popup so I can close this?

**Comment by georgemac70cr8** [comment]
no, i still have it...

![Screenshot_20200915_195413_com opibis cr8 mobile](https://user-images.githubusercontent.com/69824025/93246677-6844a500-f78d-11ea-9064-f2fbda165fa2.jpg)

**Comment by t3nshi** [comment]
Well, the measures I took before solved this on the phone that I could reproduce it before... but not in yours... so I will have to find a way to test this, of course, other than having you mailing me your phone... I'm joking of course...
I will see how I can get a phone like yours to test and fix some of the issues that only present themselves on your device model. Your phone is huawei p20... right?
